### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Pranav2734**
``<!DOCTYPE html>
<html>
<head>
  <title>Mentor Cafe</title>
  <link rel="stylesheet" type="text/css" href="style.css">
<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></script>
</head>
<body>

<header class="header">
  <nav class="navbar navbar-style">
    <div class="container" style="width: 100%; background-color: #FFFFFE; border: 2px solid #FFFFFF; overflow: hidden;position: fixed" >
      <div class="navbar-header">
        <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#micon">
          <span class="icon-bar"></span>
          <span class="icon-bar"></span>
          <span class="icon-bar"></span>
        </button>
        <a href=""></a>
      </div>
      <div class="collapse navbar-collapse" id="micon">
      <ul class="nav navbar-nav navbar-right" style="margin-right: 100px; padding-top: 20px">
        <li><a href="#Home" style="font-size: 24px; color: #FF0000;">Home</a></li>
        <li><a href="#How_It_Works" style="font-size: 24px; color: #FF0000;">How it Works</a></li>
        <li><a href="#Our_Mentors" style="font-size: 24px;color: #FF0000;" >Our Mentors</a></li>
        <li><a href="https://docs.google.com/forms/d/e/1FAIpQLSeWgpirGucJWE30KTwRckt2dDnub5OGwmvko9aWeaeIL1cbRg/viewform" style="border-radius: 25px;margin-left: 10px; width: 200px; height: 50px;background-color: #0000FF;color: #FFFFFF;font-size: 24px;text-align: center">Sign Up</a></li>
      </ul>
    </div>
    </div>
  </nav>
<div class="container-fluid" style="width: 100%%">
  <div class="row">
    
    <div class="col-sm-G"><a id="Home"></a>
      
    </div>
</div>

    <div class="col-sm-G" style="margin-left: 70px;">
      
      <p style="float: left;font-size: 27px; padding-top: 30px;margin-left: 200px">Connect with one of our curated mentors</p> 
      <input TYPE="button" VALUE="Sign Up, for free!" style="border-radius: 30px;width: 300px;height: 70px;background-color:#0000FF;color: #FFFFFF;font-size: 25px;padding-left: 15px; margin-top: 8px; margin-right: 300px; float: right;"
        onclick="window.location.href='https://docs.google.com/forms/d/e/1FAIpQLSeWgpirGucJWE30KTwRckt2dDnub5OGwmvko9aWeaeIL1cbRg/viewform'">
    </div>
<br>
<br>
<br>
<br>
<br>

<hr style="color: #0000FF">

<div class=" col-sm-G" >
  
</div>
<br>
<br>
<br>
<br>
<br>
<div class="col-sm-G">
  
</div>

<hr style="color: #0000FF">

<div class="col-sm-G"><a id="How_It_Works"></a>
  <p style="margin-left: 120px; color: #0000FF;text-decoration: none;font-size: 50px;font-weight: bold;padding: 30px;">How It Works?</p>
</div>

<div class="col-sm-G">
  
</div>
<br>
<hr style="color: #0000FF">

<div class="col-sm-G"><a id="Our_Mentors"></a>
  <p style="margin-left: 120px; color: #0000FF;text-decoration: none;font-size: 50px;font-weight: bold;padding: 30px">OUR MENTORS</p>
</div>

<div class="col-sm-G">
  
</div>
<br>

<div class="col-sm-G">
  
  <input TYPE="button" VALUE="Become a Mentor" style="border-radius: 25px;width: 300px;height: 50px;background-color:#0000FF;color: #FFFFFF;font-size: 25px;padding-bottom: 5px;padding-left: 15px; margin-left: 315px;margin-top: 5px"
        onclick="window.location.href='https://docs.google.com/forms/d/e/1FAIpQLSeWgpirGucJWE30KTwRckt2dDnub5OGwmvko9aWeaeIL1cbRg/viewform'">
</div>
<br>
<hr style="color: #0000FF">

<div class="col-sm-G"  >
	   
           
    
  <div style="width: 1550px;">
       

      <p style="height: 70px; background-color: #202020; font-size: 30px; color: #FFFFFF;  text-align: center; padding-top: 30px; width: 1550px ">
      Contact us: support@mentorcafe.in</p>
  </div>
</div>

<br>
  
</div>

</header>
</body>
</html>

### CODE REVIEW

**Comment by Pranav2734** [comment]
`*{
	margin: 0;
	padding: 0;
}
.header
{
	height: 50px;
}
.navbar-style
{
	box-shadow: 0 5px 10px #efefef;

}
.icon-bar
{
	background: red;
}`
